cirq/sim/density_matrix_simulator_test.py

```diff
@@ -586,17 +528,17 @@ def test_compute_samples_displays(dtype):
         cirq.H(b),
         cirq.X(c),
         cirq.H(c),
-        cirq.pauli_string_expectation(
+        cirq.approx_pauli_string_expectation(
```

Comment:
this won't pass (missing num_samples arg)

Reply:
fixed